llm/server.go

```diff
@@ -712,8 +714,14 @@ func (s *llmServer) Completion(ctx context.Context, req CompletionRequest, fn fu
 			}
 
 			if c.Stop {
+				doneReason := "stop"
+				if c.StoppedLimit {
+					doneReason = "length"
```

Comment:
```suggestion
					doneReason = "limit"
```

`limit` makes more sense here, and is closer to other APIs for familiarity

Reply:
Wait, `length` is right here, sorry!